runtimes/pythonrt/README.md

```diff
@@ -0,0 +1,119 @@
+# Python Runtime
+
+Implementation of Python runtime. 
+See [Python runtime](https://linear.app/autokitteh/project/python-runtime-be87fe4c4d7d) for list of issues.
+
+Currently, we don't support 3rd party packages (e.g. `pip install`) for the user code.
+See [ENG-538](https://linear.app/autokitteh/issue/ENG-538/support-python-dependencies) for more details.
+For a realistic POC/demo, we'll pre-install the packages the user code requires (e.g. `slack-sdk`).
+
+## Python Handler Function
+
+Python's handler functions should receive a single `dict` value which is the triggering event.
+This event has the following keys:
+
+- `event_type` (str): The event type
+- `event_id` (str): The event ID
+- `original_event_id` (str): The original event ID
+- `integration_id` (str): The ID of the integration that fired the event
+- `data` (dict): The event payload (integration specific)
+
+The return value from Python entry points is ignored and is not passed back to autokitteh.
```

Comment:
we should discuss the correct behavior here

Reply:
please file a ticket

Reply:
ENG-644